src/withAuthUserTokenSSR.js

```diff
@@ -93,54 +90,27 @@ const withAuthUserTokenSSR =
 
     // If specified, redirect to the login page if the user is unauthed.
     if (!AuthUser.id && whenUnauthed === AuthAction.REDIRECT_TO_LOGIN) {
-      const authRedirectDestination = authPageURL || getConfig().authPageURL
-      if (!authRedirectDestination) {
-        throw new Error(
-          `When "whenUnauthed" is set to AuthAction.REDIRECT_TO_LOGIN, "authPageURL" must be set.`
-        )
-      }
-      const destination =
-        typeof authRedirectDestination === 'string'
-          ? authRedirectDestination
-          : authRedirectDestination({ ctx, AuthUser })
-
-      if (!destination) {
-        throw new Error(
-          'The "authPageURL" must be set to a non-empty string or resolve to a non-empty string'
-        )
-      }
+      const redirect = getRedirectToLoginDestination({
+        ctx,
+        AuthUser,
+        options: withAuthUserSSRConfig,
```

Comment:
Please explicitly pass only the properties that are used (construct a new object or name each property).